documentation/docs/scenarios/upload-to-transportrequest/Readme.md

```diff
@@ -0,0 +1,197 @@
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and attach the build result to a transport request in an SAP ABAP system.
+
+There are in principle two technical ways how to attach a binary to an ABAP transport request. We support an upload via RFC and an upload via OData. Which option to use depends on the version of your ABAP system. For AS ABAP 7.50 SP08 or 7.51 SP07 or 7.52 SP03 or newer the OData based upload should be used, for older versions the RFC based upload needs to be used.
+
+## Prerequisites
+
+* [Docker environment](https://docs.docker.com/get-started/)
+* You have set up project “Piper”. See [guided tour](../../../guidedtour/)
+* A transport request
+* Depending on the version of the ABAP system: Docker image for attaching binaries to transport requests via RFC available. Due to legal reasons there is no pre-build docker image. How to create the docker image is explained [here](https://github.com/SAP/devops-docker-images/tree/master/node-rfc)
+
+### Project Prerequisites
+
+This scenario requires additional files in your project and in the execution environment on your Jenkins instance.
+
+On the project level, provide and adjust the following template:
+
+| File Name | Description | Position |
+|-----|-----|-----|
+| [`mta.yaml`](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/scenarios/rfc-upload/files/mta.yaml) | This file controls the behavior of the MTA toolset. | Place the `mta.yaml` file in your application root folder and adjust the values in brackets with your data. |
+
+Depending on the modules in your MTA, additional configuration files are required, e.g. `pom.xml`  or `package.json`.
+
+## Context
+
+This scenario combines various different steps to create a complete pipeline.
+
+In this scenario, we want to show how to build an application based on SAPUI5 or SAP Fiori by using the multi-target application (MTA) concept and how to attach the build result to a transport request inside an ABAP system. This document comprises the [mtaBuild](../../../steps/mtaBuild/) and the [transportRequestUploadFile](../../../steps/transportRequestUploadFile/) steps.
+
+In case of an RFC basd upload the binary is not streamed to the ABAP endpoint. Instead an URL pointing to the binary needs to be provided. Hence the binary must be published first so that it can be accessed via HTTP. This can happen by uploading the binary to a blob store or by archiving the artifact on Jenkins. The corresponding URL needs to be provided when the artifact is attached to the transport request.
+
+The transport request can be created on the fly (see [transportRequestCreate](../../../steps/transportRequestCreate/)) or we can use an already existing transport request. In case we use an already existing transport request Id the transport request Id needs to be provided in the git commit history (see example below) or the transport request id needs to be provided inside the job (e.g. as a job parameter).
```

Comment:
punctuation missing, inconsistent Id or id

Reply:
Changed from `id` to `Id`. Typo (based) fix. I can't find a punctuation issue.